test/e2e/Tests/Tests/HTTPFeatureTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
+
+[Collection(Constants.FunctionAppCollectionName)]
+public class HTTPFeatureTests
+{
+    private readonly FunctionAppFixture _fixture;
+    private readonly ITestOutputHelper _output;
```

Comment:
Looks like this code is not following our style guidelines. For example:
```suggestion
    readonly ITestOutputHelper output;
```
Please update accordingly.